contents/handbook/small-teams/experimentation/objectives.mdx

```diff
@@ -1,35 +1,48 @@
-    - (stretch goal) CI implementation that allows people to check flags in code deployment against what's active + exists
+### Objective: Increase deployability of Feature Flags
 
 * **Rationale:**
-    * We want feature flags to be used across larger engineering teams. Currently we aren't solving their needs fully but we have a clear path to doing so.
-
-### Objective: Launch Direct SQL Querying Beta
+     * Feature flags is a high friction feature. Unlike adding a snippet or simple line to track an event, engineers and product owners need to plan for their product to work with the flag on and off. 
+     To fully enable engineers to build with feature flags, we should make it remarkably easy to create flags.
+ 
+- **Key Result:** Activation rate of users using flags by 20% (Base metric TBD)
+- **Key Result:** TBD
+    
+Possible avenues:
+- Auto detection of feature flags being sent that don't have a corresponding flag in posthog and easy creation (like github asking for branch to PR)
+- API Enablement. Create experiments or feature flags using posthog API.
+- Better hook and time to deployment with feature flag creation when someone lands on the feature flag page
+- Less related but important: JSON flags, feature flag resiliency
 
-* **Key Result:**
-     * Five beta users who retain for three weeks
-* **Key Result:**
-     * Understand the possible product directions to extend direct sql querying
+### Objective: Understand similarities between users throughout posthog
 
 * **Rationale:**
-     * To fully support our technical users, we need to allow them to ask any sort of question about their data that might not be fit for an interface
+     * Help engineers stay focused on building. Give them as close to the answers and insights as possible with ample proof and dropoff points for further exploration.
  
+- **Key Result:** Internal discoveries around different segments of users and groups
+- **Key Result:** 
+
+ Possible avenues:
+- Cohort correlation. What are property and behavioral similarities between users in this cohort
+- Experiment correlation. What are property and behavioral similarities between successful and failing variants
+
+
+### Objective: Nail feature analysis
+
+* **Rationale:**
+     * We have internal power users on the growth team who actively find value in deploying with experiments. Making the act of quick insights for features seamless sets us up to deliver an opinionated workflow
+
+- **Key Result:** Our growth team is able to ship and draw conclusions painlessly on every feature being tested
+- **Key Result:** TBD
+
+Possible avenues:
+- Complete visibility into who's seeing a flag enabled feature. See recordings connected with users
+- Clarity around how users are bucketed in experiments. How do release conditions relate to control vs variants and how do variant overrides relate to those?
+- Experiments UX (pushed from last Q)
```

Comment:
Top things I'd like to have, in no specific order:
- Edit the experiment after it starts
- Restart the experiment (pause, edit, restart)
- See full insights of secondary metrics (and why only 3?)
- Change the metrics without restarting the experiment
- Easily create an insight filtered by the current feature flag, with a breakdown for all variants
- Easily edit the feature flag from the experiment page

I'm sure @raquelmsmith has a ton more :D